server/src/main/java/com/flipkart/varadhi/RestVerticle.java

```diff
@@ -62,14 +68,15 @@ public void start(Promise<Void> startPromise) {
         options.setUseAlpn(true);
 
         // TODO: create config for http server
-        httpServer = vertx.createHttpServer(options).requestHandler(router).listen(8080, h -> {
-            if (h.succeeded()) {
-                log.info("HttpServer Started.");
-            } else {
-                log.warn("HttpServer Started Failed.");
-            }
-            startPromise.handle(h.map((Void) null));
-        });
+        httpServer =
```

Comment:
See if we can move HttpServerOptions itself to Config ?

Reply:
that was the intention of creating `varadhiDeploymentConfig` , We can extend that to HTTP config, but for testing purpose not required I think so

Reply:
only thing was our object would be more extensible, but looking at default httpserverOption, it itself is quite vast with options, have refactored